### DISCUSSION TITLE
Angular Deviation between loading direction and <100> direction in Cubic system

### DISCUSSION THREAD

**Discussion by WenxiLee** [mtex-toolbox/mtex#2019] [Ask Anything] [answered]
Hello,

I'm attempting to compute the angular distance between the loading direction and the <100> pole in an inverse pole figure. My approach involves converting the loading direction y-vector into the crystal frame and determining the angle between the loading vector in the crystal frame and the <100> direction within the crystal frame using crystal symmetry. However, I've encountered some angles exceeding 45 degrees, which is confusing because as one <100> moves away, another <100> should move closer within the 45-degree range. I've included my code below. Could someone assist in explaining this? Thank you.

cs = crystalSymmetry('cubic');

ori = orientation.rand(1000,cs);

r= Miller(0,0,1,cs,'uvw')

Y_crystal=inv(ori) * yvector   % convert the loading direction yvector into the crystal frame

%%
angle_a=[];angle_b=[];
angle_a=angle(r,Y_crystal) / degree;
angle_b=angle(r,Y_crystal,'antipodal') / degree;

**Comment by kilir** [comment, marked as answer]
Hi,
what is the angle between a <001> and a direction pointing towards a <1,1,1>?  If I understand your task correctly, the assumption of angles being confined to <=45 degree is not correct.

Hope that helps.

Cheers,

Rüdiger
